src/ConnectedMode/QualityProfiles/QualityProfileDownloader.cs

```diff
@@ -94,19 +93,19 @@ public async Task<bool> UpdateAsync(BoundSonarQubeProject boundProject, IProgres
 
             EnsureProfilesExistForAllSupportedLanguages(boundProject);
 
+            var qpToUpdate = await outOfDateQualityProfileFinder.GetAsync(boundProject, cancellationToken);
```

Comment:
Minor: the variable name `qpToUpdate` suggests a single QP, not a list.

Maybe something like `profilesToUpdate` / `outOfDateProfiles`  instead?